politeiawww/backend.go

```diff
@@ -670,7 +671,9 @@ func (b *backend) validateProposal(np www.NewProposal, user *database.User) erro
 				imageExceedsMaxSize = true
 			}
 		} else {
-			numMDs++
+			if strings.HasSuffix(v.Name, ".md") {
```

Comment:
Shouldn't you be detecting the content type instead of just checking the file name?

Reply:
You're right @lukebp. The origin of this issue is actually the `PolicyMaxMDs` property.  Should we set that in the config file at runtime, and make it accept more than one .md file? This would solve #704 and #695.

Reply:
Hey @lukebp , as it was discussed in slack, now the policy for max MD files can be setted in config file. Its default remains 1, but one can set this to be higher and allow for proposals to accept .md and .txt attachments.